### PR TITLE
update docs to fix broken image links

### PR DESCRIPTION
**Pull request by MichaelClifford**
This should fix all the broken image links

### CODE REVIEW

File: docs/serving_monitoring/monitor-jh.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 * **How to Monitor JupyterHub workloads using Grafana dashboards** 
-    -  (Video tutorial forthcoming)
+
+[![JH how to](https://img.youtube.com/vi/1zZ-7mdxOWU/0.jpg)](https://youtu.be/1zZ-7mdxOWU)
+
     -  To troubleshoot issues while running your notebooks, you can refer to the [Troubleshooting Runbooks][12]
```
Comment (aakankshaduggal) [inline docs/serving_monitoring/monitor-jh.md:5]: This bit is not rendering properly here. 
I think this is an indentation issue.

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm 
Added a minor comment.

File: docs/serving_monitoring/monitor-jh.md (hunk 1 above)
Comment (MichaelClifford) [reply docs/serving_monitoring/monitor-jh.md]: I think I fixed it

Comment (aakankshaduggal) [reply docs/serving_monitoring/monitor-jh.md]: Thanks :+1:

**Review by aakankshaduggal — COMMENTED** [review]
/approve

**Pull request merged**